cli-copy.test.js

```diff
@@ -0,0 +1,36 @@
+// -----------------------------------------------------------------------------------------
+// #region Imports
+// -----------------------------------------------------------------------------------------
+
+const { HELP_OPTIONS } = require("./_modules/constants");
+const testUtils        = require("./tests/test-utils");
+
+// #endregion Imports
+
+// -----------------------------------------------------------------------------------------
+// #region Tests
+// -----------------------------------------------------------------------------------------
+
+describe("cli-copy", () => {
+    // -----------------------------------------------------------------------------------------
+    // #region -h, --help
+    // -----------------------------------------------------------------------------------------
+
+    describe(HELP_OPTIONS, () => {
+        test.each`
+            option
+            ${"-h"}
+            ${"--help"}
+        `("when passed '$option', it displays the help menu", async ({ option }) => {
```

Comment:
These help tests are a perfect case to use 'shared tests/specs'. Recommend creating a tests/shared sorta of directory. Using BDD style wording, `shouldDisplayHelpMenu` sort of thing along with some function currying style, you could simply pass the command.

https://medium.com/@walreyes/sharing-specs-in-jest-82864d4d5f9e

Reply:
Awesome! I had thought about doing something like this, but didn't want to generalize them too early.